tests/bpf2c_tests/expected/atomic_instruction_fetch_add_raw.c

```diff
@@ -99,12 +99,12 @@ func(void* context)
     if ((func_helpers[0].tail_call) && (r0 == 0))
 #line 23 "sample/atomic_instruction_fetch_add.c"
         return 0;
-    // EBPF_OP_JEQ_IMM pc=7 dst=r0 src=r0 offset=2 imm=0
+        // EBPF_OP_JEQ_IMM pc=7 dst=r0 src=r0 offset=2 imm=0
```

Comment:
Not sure why do we have a change in these lines?

Reply:
That's the automatic formatting requested when committing.